packages/babel-parser/test/fixtures/es2017/async-functions/await-inside-parameters-of-async-arrow-function/output.json

```diff
@@ -0,0 +1,158 @@
+{
+  "type": "File",
+  "start": 0,
+  "end": 26,
+  "loc": {
+    "start": {
+      "line": 1,
+      "column": 0
+    },
+    "end": {
+      "line": 1,
+      "column": 26
+    }
+  },
+  "errors": [
+    "SyntaxError: Await expressions are only allowed inside async functions. (1:11)",
+    "SyntaxError: Await cannot be used as name inside an async function (1:11)"
```

Comment:
In this case there are two contradicting errors in the same position, IMO it is confusing and we should fix it.